src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
Mattermost server changes the title by appending `*` when there are unread messages. So this event should be ignored when the tab is not activated. Probably `MattermostView` might need `active` prop.